docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md

```diff
@@ -0,0 +1,332 @@
+---
+id: ms-graph-azure-ad-reporting
+title: MS Graph Azure AD Reporting
+sidebar_label: MS Graph Azure AD Reporting
+description: The Sumo Logic App for MS Graph Azure AD Reporting helps you to monitor and analyze user activity, sign-in patterns and provisioning activities.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/ms-graph.svg')} alt="icon" width="50"/>
+
+The Sumo Logic App for MS Graph Azure AD Reporting enables you to access and monitor data, including audit information, user activity, sign-in patterns, and provisioning activities. It helps you to gain insights into how your organization uses Azure AD and identify potential security issues. 
+
+Key features of the Microsoft Graph Azure AD Reporting app include:
+- **Analyze Audit Activities**. Provides real-time user activity for every resource category and shows the distribution of audits by operations and services.
+- **Monitor Sign-In activities**. Monitor sign-in activities for your organization, including the number of successful and failed sign-ins, sign-ins by user location, sign-ins by application, and sign-ins from risky countries.
+- **Identify Suspicious Activities**. Identify suspicious activity, such as sign-ins from unfamiliar/risky locations or multiple sign-in attempts.
+- **Analyze Provisioning Activities**. Provides distribution of provisioning activities by status, actions, and initiators. It also shows the most frequent service principals used while provisioning.
+
+## Log Types
+
+The App uses Microsoft Graph Azure AD Reporting Source to collect [Audit](https://learn.microsoft.com/en-us/graph/api/resources/directoryaudit?view=graph-rest-1.0), [Sign-in](https://learn.microsoft.com/en-us/graph/api/resources/signin?view=graph-rest-1.0), and [Provisioning](https://learn.microsoft.com/en-us/graph/api/resources/provisioningobjectsummary?view=graph-rest-1.0) activities.
+
+## Sample Log Message
+
+<details><summary>Click here to expand</summary>
```

Comment:
```suggestion
<details><summary>View Sample Log Message</summary>
```

Reply:
I think it's better if we change the expand button text from"Click here to expand" to View Sample Log Message.